tests/samplesheet/test_long_reads.csv

```diff
@@ -0,0 +1,5 @@
+study_accession,reads_accession,fastq_1,fastq_2,library_layout,library_strategy,platform,assembler,assembly_memory,assembler_config
+ERP118188,ERR3792762,/hps/nobackup/rdf/metagenomics/service-team/users/germanab/miassembler_tests/ERR3792762.fastq.gz,,single,metagenomic,ont,,,
```

Comment:
HPS paths. If you need those files, make sure they are small and upload them to the repo (<20MB). Otherwise we need to start our own repo with test data.

Reply:
Thanks, will keep it in mind. the samplesheet is yet to be fine-tuned.